tests/testthat/test-plot_sample_hist.R

```diff
@@ -0,0 +1,19 @@
+test_that("get expected number of grobs", {
```

Comment:
Same as above. Just wrap everything in one function. Then I think everything is fine.